src/main/java/org/openstreetmap/josm/plugins/pt_assistant/actions/IncompleteMembersDownloadThread.java

```diff
@@ -33,7 +34,7 @@ public IncompleteMembersDownloadThread() {
 
     @Override
     public void run() {
-
+        PTAssistantSnackbar downloadingSnackbar = new PTAssistantSnackbar("Downloading incomplete maps...", -1);
```

Comment:
This is user facing text, that can be translated (`I18n.tr()`)

Reply:
… You don't  need to translate it, you just need to mark it as translatable by replacing this call with:
```
new PTAssistantSnackbar(I18n.tr("Downloading incomplete maps..."), -1);
```

Reply:
@michaelzangl thanks for the clarification, that's what I meant :wink: